### ISSUE TITLE
Support cargo-script

### ISSUE THREAD

**Issue by zayenz** [intellij-rust/intellij-rust#2570]
[cargo-script](https://github.com/DanielKeep/cargo-script) is useful for writing small scripts in Rust. An example from the site is a script like the following

```rust
#!/usr/bin/env run-cargo-script
//! This is a regular crate doc comment, but it also contains a partial
//! Cargo manifest.  Note the use of a *fenced* code block, and the
//! `cargo` "language".
//!
//! ```cargo
//! [dependencies]
//! time = "0.1.25"
//! ```
extern crate time;
fn main() {
    println!("{}", time::now().rfc822z());
}
```

It would be useful if the intellij-rust plugin could recognize the embedded cargo data in the comment, and thus support developing small rust-scripts also.

**Comment by JakubAdamWieczorek** [comment]
The project looks neat but it doesn't seem to have been maintained since October 2017. That's a relatively long time in the Rust world, isn't it? 🙂

**Comment by zayenz** [comment]
Yes, it does seem that development is not very active for cargo-script (I hadn't noticed that myself). That has not stopped it from being very useful for me, unsure how used it is by others.
